File: 05 - Python/09-08-00-EjercicioPropuesto-RedondeoTruncado-ULP.py
# Funciones Incorporadas en Python
import math

# redondeo y truncado
numero = 8.5
print(f'Valor original: {numero}')
# round() - redondea al entero par mas cercano si termina en .5
# se llama Banker's Rounding (o round-to-even) (redondeo-a-un-par)
redondeo = round(numero)
print(f'Redondeo: {redondeo}')
# Truncado, elimina los decimales
truncado = math.trunc(numero)
print(f'Truncado: {truncado}')
# math.floor() redondea al entero inferior mas cercano
floor = math.floor(numero)
print(f'Floor: {floor}')
# math.ceil() rendodea al entero superior mas cercano
ceil = math.ceil(numero)
print(f'Ceil: {ceil}')

File: 02 - C/09-08-00-EjercicioPropuesto-RedondeoTruncado-ULP.c
#include<stdio.h>
#include<math.h>

int main() {
	float numero;
	int redondeado, truncado;
	// Redondeo de numero
	numero = 8.6;
	// round -> Mayor a .5 redondea hacia arriba
	redondeado = round(numero);
	printf("Valor %.2f redondeado = %d\n", numero, redondeado);
	truncado = trunc(numero);
	printf("Valor %.2f truncado = %d\n", numero, truncado);
	return 0;
}

